RecoBTag/TensorFlow/python/pfNegativeDeepFlavourTagInfos_cfi.py

```diff
@@ -3,5 +3,7 @@
 from RecoBTag.TensorFlow.pfDeepFlavourTagInfos_cfi import pfDeepFlavourTagInfos
 
 pfNegativeDeepFlavourTagInfos = pfDeepFlavourTagInfos.clone(
-        shallow_tag_infos = cms.InputTag('pfDeepCSVNegativeTagInfos')
-        )
+    shallow_tag_infos = cms.InputTag('pfDeepCSVNegativeTagInfos'),
+    secondary_vertices = cms.InputTag('inclusiveCandidateNegativeSecondaryVertices'),
+    flip = cms.bool(True)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
done here https://github.com/cms-sw/cmssw/commit/b62675266e3129ca3f77a62f2d7b29a8546d6f84#diff-9e547e1a61fc38bcd4dc1951e6eb618fR6